extras/create_send_notifications.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+# comment parts that generate and mail a report
```

Comment:
What is this line?  Did you mean to commit it?

Reply:
No. I thought it was in there initially but I probably added it.

Reply:
Fixed in https://github.com/cisagov/cyhy-reports/pull/96/commits/cf0445edb96b7645e7996d962f0819b296bb8cb1

Reply:
@jasonodoom - In a case like this where someone suggests a change and you make it, we generally give the suggester credit by adding a `Co-authored-by` line to the commit message à la `Co-authored-by: Shane Frasier <jeremy.frasier@gwe.cisa.dhs.gov>` (as in commit 654c155).